docs/ONBOARDING.md

```diff
@@ -41,15 +43,23 @@ Once you are part of the twindle team, please join us here to keep in touch.
 - If you are using terminal(Mac) or command prompt(windows), watch this step by step video:
 
 [![tutorial](https://img.youtube.com/vi/7I9StcZt5cI/0.jpg)](https://youtu.be/7I9StcZt5cI).
+<br><br>
```

Comment:
for future reference 2 enter is treated as br

Reply:
yeah, for some reason it wasn't showing too much space in the preview, so when I added 2, it worked.

Reply:
modified.